chart/templates/rbac.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: 'mission-control-reader'
+rules:
+  - apiGroups: ['']
+    resources: ['configmaps', 'secrets']
+    verbs: ['get', 'list', 'watch', 'create', 'update', 'patch', 'delete']
+  - apiGroups: ['apiextensions.k8s.io']
+    resources: ['customresourcedefinitions']
+    verbs: ['get', 'list', 'watch', 'create', 'update', 'patch', 'delete']
```

Comment:
This shouldn't be allowed - It is the canary-checker, mission-control and config-db apigroups / resources that need to be included